docs/executor/index.rst

```diff
@@ -0,0 +1,30 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Executor
+========
+
+Executors are the mechanism by which task instances get run.
+
+Airflow has support for various executors. Current used is determined by the ``executor`` option in the ``core``
+section of the configuration file.
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
+
+    *
```

Comment:
Is this purposefully or a typo?

Reply:
It works - show all files from current directory except index.
Other usage of this mechanism:
https://github.com/apache/airflow/blob/master/docs/howto/operator/gcp/index.rst

Reply:
👍